Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM alpine
+
+ADD bin /app/bin
+ADD gdrive-userscript /app/gdrive-userscript
+ADD player /app/player
+ADD src /app/src
+ADD templates /app/templates
+ADD www /app/www
+ADD .eslintrc.js /app/.eslintrc.js
+ADD index.js /app/index.js
+ADD package.json /app/package.json
+ADD package-lock.json /app/package-lock.json
+ADD postinstall.sh /app/postinstall.sh
+ADD servcmd.sh.js /app/servcmd.sh.js
+ADD container-install.sh /app/container-install.sh
+
+RUN cd app && sh container-install.sh
+
+ENV MYSQL_HOST localhost
+ENV MYSQL_PORT 3306
+ENV MYSQL_DATABASE cytube
+ENV MYSQL_USER cytube
+ENV MYSQL_PASSWORD nico_best_girl
```

Comment:
These are placeholder values, since docker does not allow the ENV command to have empty values. The user should set these values using the `-e` flag when starting the container. Building the container should be a one-time thing that is done by a CI task